core-library/src/main/java/org/silverpeas/core/socialnetwork/relationship/RelationShipDao.java

```diff
@@ -223,15 +211,16 @@ public boolean isInRelationShip(Connection connection, int user1Id, int user2Id)
   }
 
   /**
-   * Get list socialInformationRelationShip (relationShips) of my Contacts according to number of
-   * Item and the first Index
-   * @param con
-   * @param myId
-   * @param myContactsIds
-   * @param begin
-   * @param end
-   * @return List<SocialInformationRelationShip>
-   * @throws SQLException
+   * Gets from the data source all the social information that were emitted between the two
+   * specified dates and that are about the relationships of the contacts of the specified
+   * user.
+   * @param con the connection to the data source.
+   * @param myId the unique identifier of a user in Silverpeas.
+   * @param myContactsIds a list of unique identifiers of the contacts of the user.
+   * @param begin the begin date of the search interval.
+   * @param end the end date of the search interval.
+   * @return a list of {@link SocialInformationRelationShip} instances.
+   * @throws SQLException if an error occurs while gettings the social information.
    */
   List<SocialInformation> getAllRelationShipsOfMyContact(Connection con,
       String myId, List<String> myContactsIds, Date begin, Date end) throws
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "myId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1172)